src/envo/modules/entity_attribute.csv

```diff
@@ -24,7 +24,7 @@ ENVO:09200027,,PATO:0000918,volume,ENVO:01001564,rain
 ENVO:09200028,,PATO:0000008,speed,ENVO:06105231,soil water infiltration
 ENVO:09200029,,PATO:0000915,thickness,ENVO:02000059,topsoil
 ENVO:09200030,,PATO:0000008,speed,ENVO:06105241,soil drainage
-ENVO:09200031,,PATO:0000080,amplitude,ENVO:01000753,air temperature
+ENVO:09200031,,PATO:0000080,amplitude,ENVO:09200001,temperature of air
```

Comment:
This term used a deprecated ENVO filler term which I replaced with the non-deprecated counterpart.

Reply:
@pbuttigieg you should double check.